Uyanda.Coffee.Application/Features/BeverageManagement/Persistence/IBeverageAccessor.cs

```diff
@@ -7,11 +7,8 @@ namespace Uyanda.Coffee.Application.Features.BeverageManagement.Persistence
 {
     public interface IBeverageAccessor
     {
-        Task<IEnumerable<BeverageModel>> AddBeveragesAsync(IEnumerable<BeverageModel> beverages);
-        Task<IEnumerable<BeverageModel>> GetBeveragesAsync(IEnumerable<BeverageModel> beverages);
-        Task<IEnumerable<AvailableCoffeeCupModel>> GetCoffeeCupsAsync();
-        Task<IEnumerable<BeverageSizeCostModel>> AddBeverageCostAsync(IEnumerable<BeverageSizeCostModel> prices);
-        Task<IEnumerable<BeverageSizeCostModel>> GetBeverageCostAsync();
+        Task<BeverageSizeCostModel[]> AddBeverageCostAsync(IEnumerable<BeverageSizeCostModel> prices);
```

Comment:
Will this also contain all the information needed to add an entirely new Beverage? As in the Beverage's name, type, . . ?